docs/internals/release-process.txt

```diff
@@ -37,30 +39,10 @@ security purposes, please see :doc:`our security policies <security>`.
 
 .. glossary::
 
-  Major release
-    Major releases (A.B, A.B+1, etc.) will happen roughly every nine months --
-    see `release process`_, below for details. These releases will contain new
-    features, improvements to existing features, and such.
-
-    .. _internal-release-deprecation-policy:
-
-    A major release may deprecate certain features from previous releases. If a
-    feature is deprecated in version ``A.B``, it will continue to work in versions
-    ``A.B`` and  ``A.B+1`` but raise warnings. It will be removed in version
-    ``A.B+2``.
-
-    So, for example, if we decided to start the deprecation of a function in
-    Django 1.7:
-
-    * Django 1.7 will contain a backwards-compatible replica of the function which
-      will raise a ``RemovedInDjango19Warning``. This warning is silent by
-      default; you can turn on display of these warnings with the ``-Wd`` option
-      of Python.
-
-    * Django 1.8 will still contain the backwards-compatible replica. This
-      warning becomes *loud* by default, and will likely be quite annoying.
-
-    * Django 1.9 will remove the feature outright.
+  Feature release
+    Feature releases (A.B, A.B+1, etc.) will happen roughly every eight months
+    -- see `release process`_, below for details. These releases will contain
+    new features, improvements to existing features, and such.
 
   Minor release
```

Comment:
or `patch` releases, it's the `semver` term and this one doesn't conflict with our historical usage.